recipes/evalhyd-cli/bld.bat

```diff
@@ -0,0 +1,12 @@
+cmake -G "Ninja" ^
+      -B build/ ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_INSTALL_LIBDIR=lib ^
+      -D CMAKE_BUILD_TYPE=Release
+if %ERRORLEVEL% neq 0 exit 1
+
+cmake --build build/ --parallel %CPU_COUNT%
```

Comment:
Is there a `make test` or `ctest` we can run in these cmake builds?

Reply:
No, the unittests for this CLI are actually written in Python (calling the executable). Should these be used in the "test" section of the recipe?